.github/ISSUE_TEMPLATE/pull_request_template_berk_schema.yml

```diff
@@ -0,0 +1,103 @@
+name: Berkeley Schema Change
+description: Describe all PRs for changes during berkeley-schema freeze
+title: "[berkeley-schema] <title>"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Guidelines
+        
+        ## _Light_ Schema Freeze
+        The nmdc-schema and berkeley-schema are under a light freeze this means changes **should not** be made that have any down stream implications. 
+        To ensure this, PRs need to be closely reviewed with **every** component of the NMDC in mind.
+        
+        ## Reviewers
+        To ensure no changes are made unexpectedly, ALL berkeley-schema refacror roll out task coordinators should be listed as reviewrs for ALL PRs. 
+        See [squad proposal](https://docs.google.com/document/d/1XXN1YuaBuSkxPXeiLKm5YxYzXTamBPQrzzeLhlh7PWs/edit#heading=h.u52g8v319adh) for list of names.
+        PRs should NOT be merged until they have been approved by all task coordinators.
```

Comment:
What commitment are the task leaders making to punctually review PRs?

Reply:
Is this an issue template a PR template or both?

Reply:
It's a PR template but far as I could tell, they all go into the ISSUE_TEMPLATE folder. but I'm gonna have to deploy it and check!

Reply:
made PR template directory